examples/device/cdc_msc/src/main.c

```diff
@@ -137,13 +137,24 @@ void tud_cdc_line_state_cb(uint8_t itf, bool dtr, bool rts)
   {
     // print initial message when connected
     tud_cdc_write_str("\r\nTinyUSB CDC MSC device example\r\n");
+    tud_cdc_write_flush();
   }
 }
 
 // Invoked when CDC interface received data from host
 void tud_cdc_rx_cb(uint8_t itf)
 {
   (void) itf;
+  uint8_t const line_state = tud_cdc_get_line_state();
+
+  // Provide information that terminal did not set DTR bit
+  if( !(line_state & 0x01) )
```

Comment:
It is not necessary for example to send out the warning. I will do a follow up clean up